chain/client/src/client_actor.rs

```diff
@@ -1067,7 +1068,10 @@ impl ClientActor {
             }
 
             self.info_helper.block_processed(gas_used, chunks_in_block as u64);
+            let block_hash = block.header().hash().clone();
+            let chunks = ChunksDelayTracker::get_chunks(block);
             self.check_send_announce_account(last_final_hash);
+            self.client.record_accepted_block(&block_hash, &chunks);
```

Comment:
on one side - I'd rather have this code directly in 'client' (rather than client actor)..

Reply:
Moved this into `on_block_accepted()`